.github/workflows/main.yml

```diff
@@ -0,0 +1,48 @@
+name: RHVoice-CI
+
+on:
+  push:
+    branches:
+      - 'master'
+
+jobs:
+  androidBuild:
+    name: Build android app
+    runs-on: ubuntu-18.04
+
+    steps:
+      - uses: actions/checkout@v1
+      - name: Set up JDK 1.8
+        uses: actions/setup-java@v1
+        with:
+          java-version: 1.8
+      - name: Assemble app debug APK
+        working-directory: ./src/android
+        run: sudo ./gradlew assembleDebug --stacktrace
+
+      - name: Upload app APK
+        uses: actions/upload-artifact@v1
+        with:
+          name: app-debug
+          path: src/android/RHVoice-core/build/outputs/apk/debug/RHVoice-core-debug.apk
```

Comment:
if you specified running dir for the whole job you can use Relative path.

Reply:
No, working-directory is not compatible with actions, so this change is not possible

Reply:
are you sure?

Reply:
yes, at least i tried and it didn't work and i've read on github.community that it is not possible to use working-directory in actions

Reply:
ok.